app/config/errors.php

```diff
@@ -408,6 +408,16 @@
         'description' => 'Document with the requested ID already exists.',
         'code' => 409,
     ],
+    Exception::DOCUMENT_UPDATE_CONFLICT => [
```

Comment:
Is this error code supposed to be here? Seems like its something from offline support

Reply:
we added already some metadata needed for offline support, so we can release it without a appwrite release